src/mgmt/srv_target.c

```diff
@@ -795,6 +795,25 @@ ds_mgmt_tgt_profile_hdlr(crt_rpc_t *rpc)
 	crt_reply_send(rpc);
 }
 
+/**
+ * Do Mark on a single target.
+ */
+void
+ds_mgmt_tgt_mark_hdlr(crt_rpc_t *rpc)
+{
+	struct mgmt_mark_in	*in;
+	struct mgmt_mark_out	*out;
+
+	in = crt_req_get(rpc);
+	D_ASSERT(in != NULL);
+
+	D_DEBUG(DB_TRACE, "Mark trace %s.\n", in->m_mark);
```

Comment:
If this is to be used for marking test boundaries, we might want a high log level.

Reply:
probably this mask should be an input parameter. but export DB_X seems not a nice thing to do? any suggestion?

Reply:
Maybe just using D_CRIT(...), as when user call that API it want that mark msg shows there.